src/core/pointcloud/qgspointcloudindex.h

```diff
@@ -262,7 +262,7 @@ class CORE_EXPORT QgsPointCloudIndex: public QObject
     QgsDoubleRange nodeZRange( const IndexedPointCloudNode &node ) const;
 
     //! Returns node's error in map units (used to determine in whether the node has enough detail for the current view)
-    float nodeError( const IndexedPointCloudNode &n ) const;
+    virtual float nodeError( const IndexedPointCloudNode &n ) const;
```

Comment:
why is this changed to virtual?

Reply:
I was going to use the span from COPC at some point to calculate node error and forgot to remove "virtual"